code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -33,19 +33,19 @@
 	var/need_mob_update = FALSE
 	switch(affected_mob.stat)
 		if(CONSCIOUS) //bad
-			thou_shall_heal = death_is_coming/50
-			need_mob_update += affected_mob.adjustOxyLoss(2 * REM * seconds_per_tick, TRUE, required_biotype = affected_biotype, required_respiration_type = affected_respiration_type)
+			thou_shall_heal = min(death_is_coming/20, 3)
+			need_mob_update += affected_mob.adjustOxyLoss(3 * REM * seconds_per_tick, TRUE, required_biotype = affected_biotype, required_respiration_type = affected_respiration_type)
 		if(SOFT_CRIT) //meh convert
-			thou_shall_heal = round(death_is_coming/47,0.1)
-			need_mob_update += affected_mob.adjustOxyLoss(1 * REM * seconds_per_tick, TRUE, required_biotype = affected_biotype, required_respiration_type = affected_respiration_type)
+			thou_shall_heal = round(death_is_coming/13,0.1)
+			need_mob_update += affected_mob.adjustOxyLoss(2 * REM * seconds_per_tick, TRUE, required_biotype = affected_biotype, required_respiration_type = affected_respiration_type)
 		else //no convert
-			thou_shall_heal = round(death_is_coming/45, 0.1)
+			thou_shall_heal = round(death_is_coming/10, 0.1)
 			good_kind_of_healing = TRUE
 	need_mob_update += affected_mob.adjustBruteLoss(-thou_shall_heal * REM * seconds_per_tick, FALSE, required_bodytype = affected_bodytype)
 	if(need_mob_update)
 		. = UPDATE_MOB_HEALTH
 
-	if(good_kind_of_healing && !reaping && SPT_PROB(0.00005, seconds_per_tick)) //janken with the grim reaper!
+	if(!good_kind_of_healing && !reaping && SPT_PROB(0.00005, seconds_per_tick)) //janken with the grim reaper!
```

Comment:
you dont want this do you? it should only work when you are in crit. This PR makes it so it happens when you are NOT in deep crit.

Could make soft-crit also give a chance if you are really pressed about it firing not enough.

Reply:
I was actually entirely hoping it'd work out of crit given the improvements to regular healing - you keep it in you, you're getting the oxyloss and rolling the dice. If you get lucky, you get as good as a regen slime core (most likely at a time it does you no favors if you're running around loaded on it.) If you get unlucky, you're a pile on the floor. I think it's a cooler mechanic than to just reserve to the brief time in crit, but is this going too far?